docs/install/install-troubleshooting.md

```diff
@@ -1,9 +1,11 @@
-# ISLE Installation: Troubleshooting
+# ISLE: Troubleshooting Installations
 
 **Please select a topic:**
 
 - [Port Conflicts](#port-conflicts)
 - [Non-Running Docker Containers](#non-running-docker-containers)
+- [Viewing Logs in ISLE 1.2.0](#viewing-logs-in-isle-120)
```

Comment:
I would change this to something like `Viewing Logs in ISLE version 1.2.0 and greater` so it's not so static.

Reply:
Good point. Done.